wordpress-mobile/gutenberg-mobile
# Update unsupported block test data for UI tests

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/28649

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/3086

To test:
The changes can be tested locally by running the specific UI test:

- For iOS run: `npm run test:e2e:ios:local gutenberg-editor-unsupported-blocks.test`
- For Android run: `npm run test:e2e:android:local gutenberg-editor-unsupported-blocks.test`

**The PR can be also verified with the PR check that runs the full suite of UI tests.**

PR submission checklist:
